cl/recap/mergers.py

```diff
@@ -1474,15 +1475,15 @@ def merge_attachment_page_data(
         # Appellate entries with attachments don't have a main RD, transform it
         # to an attachment.
         if (
-            court.pk in appellate_court_ids
+            await appellate_court_ids.filter(pk=court.pk).aexists()
```

Comment:
This change means an additional query to the DB for each loop through the attachment_dicts, doesn't it?

Reply:
I think it was already doing a query, just one that wasn't async compatible apparently.

Reply:
Well, before it was setting the `appellate_court_ids` variable before the loop, then checking if `court.pk` was in the results during each iteration. That'd only require one query. This hits the DB for each iteration through the loop.

Reply:
> then checking if court.pk was in the results during each iteration

From what I could tell this was generating a query for each iteration.

> This hits the DB for each iteration through the loop.

I'll just move the check outside the loop.